packages/react/src/date-picker/reactDayPickerStyles.ts

```diff
@@ -1,33 +1,46 @@
 import { boxPalette, fontGrid, mapSpacing, tokens } from '../core';
-import { visuallyHiddenStyles } from '../a11y';
 import { focusStyles, highContrastOutlineStyles } from '../box';
 
 const cellSizeLarge = '3rem';
 const cellSizeSmall = '2.75rem';
 
+// Base button
+const buttonReset = {
```

Comment:
This just just be in the `buttonNextPrevious` var as it's not used elsewhere?

Reply:
Yeah, I would merge it and do less. We only have the previous and next buttons, and I don't see that changing any time soon.